var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -134,6 +141,18 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
              destination='newlibsource',
              when='+nvptx',
              fetch_options=timeout)
+    resource(name='newlib-amdgcn',
+             url='ftp://sourceware.org/pub/newlib/newlib-3.3.0.tar.gz',
+             sha256='58dd9e3eaedf519360d92d84205c3deef0b3fc286685d1c562e245914ef72c66',
+             destination='newlibsource',
+             when='+amdgcn',  # TODO: Is there an or-selection syntax?
+             fetch_options=timeout)  # to combine with newlib, above
+    resource(name='llvm-amdgcn',
+             url='https://github.com/llvm/llvm-project/archive/llvmorg-9.0.0.tar.gz',
+             sha256='7807fac25330e24e9955ca46cd855dd34bbc9cc4fdba8322366206654d1036f2',
+             destination='llvm-source',
+             when='+amdgcn',
+             fetch_options=timeout)
```

Comment:
Isn't this just llvm? Why can't we have a dependency on it?

Reply:
According to gcc 10's build instructions, it's only the binutils part of llvm, with a special mod I think.